### DISCUSSION TITLE
KLV error

### DISCUSSION THREAD

**Discussion by gismam** [All4Gis/QGISFMV#120] [Q&A] [answered]
![klv](https://user-images.githubusercontent.com/101732033/158628622-62644d0a-9d31-4cdf-92ae-d4dd98e74d07.PNG)

Hello Fran. What do you recommend when having this issue running this plugin?

**Comment by All4Gis** [comment, marked as answer]
Hi @MarcMuehleisen , check the ffmpeg path in the settings file, check this samples:

https://github.com/All4Gis/QGISFMV/blob/master/code/settings.ini
https://github.com/All4Gis/QGISFMV/blob/master/code/settings_linux_sample.ini
https://github.com/All4Gis/QGISFMV/blob/master/code/settings_win_sample.ini

When you finish with he installation and configuration, tru with the MISB sample videos

**Reply by gismam** [reply]
I checked my settings.ini and verified that the path for ffmpeg is correct. Dowloaded the windows build from gyan.dev [https://www.gyan.dev/ffmpeg/builds/](url).
Is there a specific release that i should have used?
Followed the following tip from an older issue: _"If you download directly FFMPEG on the official website. Change the name of the folder, and place it in root like : "C:\FFMPEG". Then I noticed that the folder downloaded on the website organize the folder like this : C:\FFMPEG\bin\ .exe file, so you should move all the file of the bin folder in the previous one. For me it work by following all the instruction on the video. Hope it help."_

**Reply by All4Gis** [reply]
ok, then, if you open this video for example, https://drive.google.com/drive/folders/105RWjm879-vlmWdiHO6In4C7GoeXC78F , check the Cheyenne.ts, can you open without problems in QGIS?.

The last release  1.15
https://github.com/All4Gis/QGISFMV/releases/tag/1.15

You have this version in the QGIS repo:
https://plugins.qgis.org/plugins/QGIS_FMV/

**Reply by gismam** [reply]
Sorry for not answering. I was busy with other tasks. Yes, i am able to open Cheyenne.ts with QGIS!
Does that tell you something?

**Reply by All4Gis** [reply]
So you get this error when you get this error with videos that are MISB videos?, in short, this error means that your video does not have metadata or does not follow the stanag standard.

**Reply by gismam** [reply]
Okay i see where the problem was. Tried to directly open DJI videos with Open Video file instead of creating the MISB file. Thank you very much!

**Comment by Remiel32** [comment]
Hello Fran, i have the same issue when i import the DJI M4E video into QGIS FMV(v 1.15). I have check the setting.ini like (C:\FFMPEG\bin). When i run the plugin, it weems it can find the FFMPEG. But when i import the .MP4 file, it calls "Error interpreting klv data, metadata cannot be read. : the parser did not recognize KLV data". Can you give some advice about this problem?
